duckchat/duckchat-impl/src/main/res/values/donottranslate.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) 2024 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <string name="duck_chat_title">Duck.ai</string>
+    <string name="duck_chat_settings_activity_description">AI Chat is an optional feature available at duck.ai that lets you have private conversations with popular 3rd-party AI chat models. Your chats are not used to train chat models.\n<annotation type="learn_more_link">Learn More</annotation></string>
+    <string name="duck_chat_visibility_setting">Show Duck.ai in Browser Menu</string>
```

Comment:
Will we need translations before release?

Reply:
Yes, I think we agreed to it in the [kick-off](https://app.asana.com/0/1202096681738622/1208794468983990/f). From what I remember, we make a separate branch for that after this initial version is merged, right?

Reply:
Usually we do the translations before merging by stacking the translations branch on top of this one, then merge everything when that’s complete.

Reply:
We will do translations after this is merged.